musig/README.md

```diff
@@ -98,13 +153,13 @@ There are several paths to signing:
     - Make `s = r + c * x` where `x = privkey`
 
     Output:
-    - Signature { s, R }
+    - Signature { `s`, `R` }
 
 2. Make a Schnorr signature with one aggregated key (`Multikey`), derived from multiple public keys.
-    - Create an aggregated pubkey. For more information see the [key aggregation](#key-aggregation) section.
+    - Create a `Multikey`. For more information, see the [key aggregation](#key-aggregation) section.
 
     For each party that is taking part in the signing:
-    - Call `Party::new(transcript, privkey, multikey, pubkeys)`.
+    - Call `Party::new(transcript, privkey, multikey)`.
     - Get back `PartyAwaitingPrecommitments` and a `NoncePrecommitment`.
     - Share your `NoncePrecommitment`, and receive other parties' `NoncePrecommitment`s. 
 
```

Comment:
These blank lines don't show up in the rendered version of the document. Do you intend to include section dividers here? If so, you need to make them visible; if not, please remove these blank lines.

Reply:
I did intend to have section dividers; i'll make this more explicit.